ground/src/main/res/values/strings.xml

```diff
@@ -53,6 +53,7 @@
   <string name="settings">Settings</string>
   <string name="offline_area_viewer_title">Downloaded area</string>
   <string name="offline_area_viewer_remove_button">Remove from device</string>
+  <string name="offline_area_download_error">Download Failed! Please Try Again</string>
```

Comment:
Please use sentence case instead of title case:
"Download failed. Please try again"